src/assets/translations/main.json

```diff
@@ -168,7 +167,9 @@
         "foxyBalance": "FOXy Balance",
         "withdrawals": "Withdrawals",
         "emptyWithdraws": "Pending or active withdrawals will show here.",
-        "availableDate": "available %{date}"
+        "availableDate": "available %{date}",
+        "approveToken": "Approve",
```

Comment:
This `approveToken` translation isn't used anywhere in the codebase - why are we adding it?